🐛 Fix the current value for stats

david-martin opened the pull request:
The 'current' value of each stat should be based off the most recent
value.
This change fixes a bug where the oldest value was used instead.

david-martin commented:
@aidenkeating @wei-lee Could you check this for sanity. Its a tiny change to the code, but my understanding of that block of code is bit limited.

wei-lee commented on lib/sync-metrics.js:
```diff
@@ -72,7 +72,7 @@ var aggregateData = function(metric, records) {
     }).reduce(function(reduced, groupRecords, groupKey) {
       var processedData = _.reduce(groupRecords, function(memo, groupRecord) {
-        groupRecords = _.sortBy(groupRecords, 'ts');
+        groupRecords = _.sortBy(groupRecords, 'ts').reverse();
```
I think this line should be moved up, before L74. It should be like this:

```
groupRecords = _.sortBy(groupRecords, 'ts').reverse();
var processedData = _.reduce(groupRecords, function(memo, groupRecord) {})
...

```

The idea is to have the records sorted before running the reduce. We should only need to sort it once. Not sure how we end up this code.

Also removing `.reverse()` is right. For some reason I thought the results are in descending order.

david-martin commented:
@wei-lee Good point. I've moved it outside and that triggered the bug again, so I removed the `.reverse()` when moved outside.
That seems to be the right combination now.
